galaxy_ng/app/migrations/0010_add_staging_rejected_repos.py

```diff
@@ -0,0 +1,46 @@
+from django.conf import settings
+from django.db import migrations
+
+STAGING = settings.GALAXY_API_STAGING_DISTRIBUTION_BASE_PATH
```

Comment:
Migration should never depend on project settings.

Reply:
updated